compiler/src/lib.rs

```diff
@@ -12,5 +12,5 @@
 // limitations under the License.
 
 pub mod codegen;
-pub mod prost_codegen;
+// pub mod prost_codegen;
```

Comment:
I suppose this is committed as a mistake.

Reply:
done